io/net/iperf_test.py

```diff
@@ -183,29 +184,61 @@ def nping(self):
             cmd = "nping --tcp %s -c 10" % self.peer_ip
             return process.run(cmd, verbose=False, shell=True)
 
+    def enable_irqbalance(self):
+        """
+        Enable irqbalance service on host and peer
+        """
+        cmd_start = "systemctl start irqbalance"
+        self.session.cmd(cmd_start)
+        cmd_status = "systemctl status irqbalance"
+        output = self.session.cmd(cmd_status).stdout_text.splitlines()
+        for line in output:
+            if re.search("Active: active (running)", line):
+                self.log("irqbalance service is active in peer")
+        process.system(cmd_start)
+        for line in process.system_output(cmd_status).decode("utf-8").splitlines():
+            if re.search("Active: active (running)", line):
+                self.log("irqbalance service is active in host")
+
+    def tune_rxtx_queue(self):
+        """
+        Increase rx/tx queues to 16 for test interface
+        """
+        cmd = "ethtool -L %s rx 16 tx 16" % self.peer_interface
+        output = self.session.cmd(cmd)
```

Comment:
only change values for peer interface.. and not host_interface ?

Reply:
cmd = "ethtool -L %s rx 16 tx 16" % self.iface
This part is taking care of changing the queue value for host interface